data_visualization_in_seaborn/data_visualization_in_seaborn.md

```diff
@@ -926,7 +926,7 @@ If you want a high level of flexibility in your trend line, you can achieve that
 <div class = "learn-more">
 <b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
 
-For a deeper understanding of what is meant by parametric vs. non-parametric models, see section 2.1.2 of the book [Statistical Learning](https://hastie.su.domains/ISLR2/ISLRv2_website.pdf).
+For a deeper understanding of what is meant by parametric vs. non-parametric models, see section 2.1.2 of the book [Statistical Learning](https://hastie.su.domains/ISLR2/ISLRv2_corrected_June_2023.pdf.download.html).
```

Comment:
Is there a way to access this that isn't a download link? I imagine some folks might struggle with this depending on their browser/permissions.

Reply:
Unfortunately, no I can't find any other links directly to the current edition of the textbook other than this download link. This is also the only option available from the StatLearning website, other than links to purchase a physical copy of the book.  

I wonder if they're not allowed to host an online version of the book, just a download link only, because of agreements with the publisher or something. I agree that it's way less convenient.

Reply:
Eh I think it's fine, I just like to avoid them if possible but sometimes that's all we got! We can always revisit if we get complaints.